Sources/StreamChatUI/Common Views/ChatChannelCreateNewButton_Tests.swift

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelCreateNewButton_Tests: XCTestCase {
+    let containerView: UIView = {
```

Comment:
Hmm I would swear I wrote and published this comment.

The question is... Do we need the containerView?  Why don't we just snapshot the views created inside the tests? IMHO this is redundant.. Or not?

Reply:
I also don't get this

Reply:
This was more to see the change of size of the button. Without the container view it is not possible to understand that the button has successfully changed size. Should I remove it anyways?